graphql/resolvers/category/mutations.js

```diff
@@ -0,0 +1,31 @@
+const Category = use('App/Models/Category')
+
+const mutations = {
+  async createCategory (parent, args) {
+    const category = new Category()
+
+    category.name = args.name
+    category.description = args.description
+    await category.save()
+
+    return category
+  },
+
+  async updateCategory (parent, args) {
+    const category = await Category.findBy('id', args.id)
+
+    category.name = args.name ? args.name : category.name
+    category.description = args.description ? args.description : category.description
+    category.save()
```

Comment:
why do you have to check null or not? 

if you do a check like that, it should be like this

```suggestion
    if (args.name) {
     category.name = args.name
    }
    if (args.description) {
     category.description = args.description
    }
    category.save()
```

Reply:
because when updating, either name or description can be null. It means the client doesn't want to update all values.

Reply:
I think there is no difference Kak between the first and second codes (besides number of lines)?

Reply:
there is a difference in efficient query, for example the previous syntax will update the set even though the data is not updated, if the one below it will not update if nothing is updated, then it is easier to read too

Reply:
Ohh I see, okay thanks kak! I will revise it.